README.md

```diff
@@ -81,7 +81,7 @@ This package includes the following version of the required dependencies:
 | @actions/github | 5.0.3            |
 | @octokit/action | 4.0.4            |
 | parse-diff      | 0.9.0            |
-| simple-git      | 3.15.1           |
+| simple-git      | 3.17.0           |
```

Comment:
Not relevant to this line, but can we update the readme to tell users to install the scanner plugin via `npx` instead of  `node_modules`?